benches/oneshot/sort.rs

```diff
@@ -45,9 +45,9 @@ async fn main() -> Result<(), BoxError> {
 
     let start = Instant::now();
     let result = try_join_all(vec![
-        GenerateSortPermutation::new(&shares[0], num_bits).execute(ctx0),
-        GenerateSortPermutation::new(&shares[1], num_bits).execute(ctx1),
-        GenerateSortPermutation::new(&shares[2], num_bits).execute(ctx2),
+        generate_sort_permutation(ctx0, &shares[0], num_bits),
+        generate_sort_permutation(ctx1, &shares[1], num_bits),
+        generate_sort_permutation(ctx2, &shares[2], num_bits),
```

Comment:
By the way, I like this move to a simple function rather than an object.  It is nicer to use by far.

I do think that maybe you could make the name shorter.  Even `generate_permutation` would be less of a mouthful.

Reply:
Taken care in #265